ldes-server-port-ingest/pom.xml

```diff
@@ -40,6 +47,12 @@
             <groupId>org.mockito</groupId>
             <artifactId>mockito-junit-jupiter</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-autoconfigure</artifactId>
+            <version>3.0.6</version>
+            <scope>compile</scope>
```

Comment:
any particular reason to use compile here?

Reply:
I added this dependency with an intellij shortcut and forgot to cleanup the pom :smile: